music_assistant/providers/plex/__init__.py

```diff
@@ -944,9 +963,19 @@ async def on_streamed(
         """Handle callback when an item completed streaming."""
 
         def mark_played() -> None:
```

Comment:
I think what you are interested in is the scrobbling event to report what is playing. Have a look at the lastfm scrobbler for an example (or the template plugin provider).